dm/dm/config/task.go

```diff
@@ -346,6 +346,11 @@ type TaskConfig struct {
 
 	// extra config when target db is TiDB
 	TiDB *TiDBExtraConfig `yaml:"tidb" toml:"tidb" json:"tidb"`
+
+	// task experimental configs
+	Experimental struct {
+		AsyncCheckpointFlush bool `yaml:"async-checkpoint-flush" toml:"async-checkpoint-flush" json:"async-checkpoint-flush"`
```

Comment:
Just a disscuss: I prefer not to add an `experimental` config item, because if after GA, we need mark it as deprecated and `config export` may have two `async-checkpoint-flush` entry, which is so confused.

Reply:
We can still add deprecated comment for experimental items. After this feature is GA, I think we can just remote this config and always use async flush.

Reply:
/cc @lance6716

Reply:
I agree that we can always turn on async flush after GA